src/styles/main.scss

```diff
@@ -1 +1,62 @@
-// put code here
+body {
+  margin: 0;
+}
+
+.circle {
+  position: relative;
+  margin: 8vmin auto;
+  height: 80vmin;
+  width: 80vmin;
+  border-radius: 50%;
+  border: 2vmin dotted black;
+
+  &__point {
+    position: absolute;
+    z-index: 2;
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 50%;
+    background-color: orange;
+    margin: 37.5vmin;
+
+  }
+
+  &__bigHand {
+    z-index: 1;
+    position: absolute;
+    height: 40vmin;
+    width: 2vmin;
+    background-color: green;
+    margin: 0 39vmin;
+
+    animation-name: move;
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
+    transform-origin: bottom;
+    animation-timing-function: linear;
```

Comment:
All animation properties you can merge in one: `animation`: https://developer.mozilla.org/en-US/docs/Web/CSS/animation